.github/workflows/pull_request.yml

```diff
@@ -53,22 +53,13 @@ jobs:
 
   wasm-build:
     name: Wasm Build
-    runs-on: ubuntu-latest
-    container:
-      image: swift:6.1-noble
-    steps:
-      - name: Checkout repo
-        uses: actions/checkout@v4
-      - name: Install Swift SDKs for WebAssembly
-        run: |
-          # TODO: We can replace these Swift SDKs with the swift.org one once it supports Foundation.
-          swift sdk install https://github.com/swiftwasm/swift/releases/download/swift-wasm-6.1-RELEASE/swift-wasm-6.1-RELEASE-wasm32-unknown-wasi.artifactbundle.zip --checksum 7550b4c77a55f4b637c376f5d192f297fe185607003a6212ad608276928db992
-          swift sdk install https://github.com/swiftwasm/swift/releases/download/swift-wasm-6.1-RELEASE/swift-wasm-6.1-RELEASE-wasm32-unknown-wasip1-threads.artifactbundle.zip --checksum 0dd273be28741f8e1eb00682c39bdc956361ed24b5572e183dd8a4e9d1c5f6ec
-          swift sdk list
-      - name: Build
-        run: |
-          swift build --swift-sdk wasm32-unknown-wasi --target ArgumentParser
-          swift build --swift-sdk wasm32-unknown-wasip1-threads --target ArgumentParser
+    uses: swiftlang/github-workflows/.github/workflows/swift_package_test.yml@main
+    with:
+      enable_wasm_sdk_build: true
```

Comment:
Should I add `wasm_sdk_versions: ["nightly-main"]` or `wasm_sdk_versions: ["nightly-main", "nightly-6.2"]` too? The default value of `wasm_sdk_versions` is `["nightly-6.2"]`.

Reply:
The current `tests` job is using the default Swift versions, so I don't think I need to change `wasm-sdk_versions` in the `wasm-build` job too. Either way, `nightly-main` will likely be added to the default value. https://github.com/swiftlang/github-workflows/pull/150